contracts/tests/HotfixClaimRevenue.t.sol

```diff
@@ -0,0 +1,334 @@
+pragma solidity 0.8.16;
+
+import "forge-std/Test.sol";
+import { Denominations } from "chainlink/Denominations.sol";
+
+import { Spigot } from "../modules/spigot/Spigot.sol";
+import { Escrow } from "../modules/escrow/Escrow.sol";
+import { SecuredLine } from "../modules/credit/SecuredLine.sol";
+import { ILineOfCredit } from "../interfaces/ILineOfCredit.sol";
+import { ISecuredLine } from "../interfaces/ISecuredLine.sol";
+import { IEscrow } from "../interfaces/IEscrow.sol";
+import { ISpigot } from "../interfaces/ISpigot.sol";
+
+import {LineFactory} from "../modules/factories/LineFactory.sol";
+import {ModuleFactory} from "../modules/factories/ModuleFactory.sol";
+import {ILineFactory} from "../interfaces/ILineFactory.sol";
+
+import { LineLib } from "../utils/LineLib.sol";
+import { MutualConsent } from "../utils/MutualConsent.sol";
+
+import { MockLine } from "../mock/MockLine.sol";
+import { SimpleOracle } from "../mock/SimpleOracle.sol";
+import { RevenueToken } from "../mock/RevenueToken.sol";
+
+import { IERC20 } from "openzeppelin/token/ERC20/IERC20.sol";
+
+contract HotfixClaimRevenueTest is Test {
```

Comment:
Can u add a bunch of comments up here on why we're testing this etc. Link to where the final report will be in Notion (public link) https://debtdao.notion.site/Spigot-Claim-Revenue-Accounting-01153e95f1be47d194ec9f252304855b